packages/siop-oid4vp/lib/authorization-response/Payload.ts

```diff
@@ -24,6 +24,7 @@ export const createResponsePayload = async (
     ...(responseOpts.accessToken && { access_token: responseOpts.accessToken, expires_in: responseOpts.expiresIn || 3600 }),
     ...(responseOpts.tokenType && { token_type: responseOpts.tokenType }),
     ...(responseOpts.refreshToken && { refresh_token: responseOpts.refreshToken }),
+    ...(responseOpts.isFirstParty && { is_first_party: responseOpts.isFirstParty }),
```

Comment:
Why are we putting this in a response? Is that according to spec?

Reply:
as discussed, this will now be an option on the RP